dashboard/test/models/pd/workshop_test.rb

```diff
@@ -1492,6 +1492,11 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     assert workshop.valid?
   end
 
+  test 'Build Your Own Workshop type is valid' do
```

Comment:
I'm curious what this is testing? As is it seems like it's testing the factory itself?

Reply:
Just testing that it's possible to make a valid BYO workshop (pass the backend validation). But if I'm going to move the other test to use these params, I can delete this one.